docs/apache-airflow/migrations-ref.rst

```diff
@@ -27,7 +27,9 @@ Here's the list of all the Database Migrations that are executed via when you ru
 +---------------------------------+-------------------+-------------------+--------------------------------------------------------------+
 | Revision ID                     | Revises ID        | Airflow Version   | Description                                                  |
 +=================================+===================+===================+==============================================================+
-| ``0038cd0c28b4`` (head)         | ``44b7034f6bdc``  | ``2.4.0``         | Add Dataset model                                            |
+| ``38d6c28f9cf9`` (head)         | ``0038cd0c28b4``  | ``2.4.0``         | Add dag_owner_attributes table.                              |
```

Comment:
I probably made mistakes with the revisions and the file name of the migration script :)
Will like someone to double-check me :D

Reply:
Isn't pre-commit double-checking your :) ?

Reply:
Yep, pre-commit generated this file, and it looks good :)
What I'm not sure at is the value (that I put) as the revision ID.. I put there the last git revision of the `main` branch when I started working on this feature, but it's no longer relevant, probably :)
What is the best practice here? Setting the revision of the first commit of this branch? meaning `c449b861c7c1ff8bb02fa7ffe0bee84e90af510e`? And if so - what do you guys do - 12 last chars, or first?

Another question is about the file name of the migration script - I set the version there as 2.4.0, like the one before my added file, but I don't really know what to set as the affected version..

Reply:
No idea - I have never written a single migration - but I believe `alembic` does all for you. There is a section about it CONTRIBUTING - I am not sure how good it is - re version - yes 2.4.0 is good - we can change it afterwards.

Reply:
My bad, I completely missed this part in the CONTRIBUTING doc.. I regenerated the file using `alembic` and it created the revisions for me, you're totally right! :)

thanks! 😄